test/testSetupFile.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-env jasmine */
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
```

Comment:
you can use [`jest.setTimeout`](https://facebook.github.io/jest/docs/en/jest-object.html#jestsettimeouttimeout)

Reply:
Fixed